.cirun.yml

```diff
@@ -7,22 +7,20 @@ runners:
     instance_type: g4dn.xlarge  # Cheapest VM on AWS
     # Amazon AMI: Deep Learning AMI GPU CUDA 11.2.1 (Ubuntu 20.04) 20210625
     # for eu-west-1
-    machine_image: ami-07b138161bedd526d
+    machine_image: ami-00ac0c28c01352e53
```

Comment:
Could you find the name and region for the new AMI and update the comment above? Note that we need the AMI for eu-west-1 (I think).